### DISCUSSION TITLE
How to support being found with custom domain bluesky handle?

### DISCUSSION THREAD

**Discussion by goellner** [kawamataryo/sky-follower-bridge#104] [Q&A]
Anyone knows if there is a way to help being found with a custom domain? 

Is there maybe any phrase or format to put in the display name or bio, which could support the bridge?

example: 
`🦋 (@wesbos.com)`

would make it easy to be supported by a regex, which makes it easy to extract

**Comment by kawamataryo** [comment]
Thank you for your question. If you include a link to your Bluesky profile in your bio on X, it can be detected from that link. Here is an example of such a bio.

<img width="727" alt="image" src="https://github.com/user-attachments/assets/96ab788f-d177-4408-961e-1e70f8b0e21d">
